Extension/i18n/chs/package.i18n.json

```diff
@@ -70,35 +70,35 @@
 	"c_cpp.configuration.suggestSnippets.description": "如果为 true，则由语言服务器提供片段。",
 	"c_cpp.configuration.enhancedColorization.description": "如果启用了 IntelliSense，则基于 IntelliSense 对代码进行着色。如果禁用了 IntelliSense 或使用默认高对比度主题，则此设置不起任何作用。",
 	"c_cpp.configuration.vcpkg.enabled.markdownDescription": "为 [vcpkg 依赖关系管理器] 启用集成服务(https://aka.ms/vcpkg/)。",
-	"c_cpp.debuggers.pipeTransport.description": "如果存在，这会指示调试程序使用其他可执行文件作为管道来连接到远程计算机，此管道将在 VS Code 和已启用 MI 的调试程序后端可执行文件(如 gdb)之间中继标准输入/输入。",
-	"c_cpp.debuggers.pipeTransport.default.pipeProgram": "输入管道程序名称的完全限定路径，例如 \"/usr/bin/ssh\"",
+	"c_cpp.debuggers.PipeTransport.description": "如果存在，这会指示调试程序使用其他可执行文件作为管道来连接到远程计算机，此管道将在 VS Code 和已启用 MI 的调试程序后端可执行文件(如 gdb)之间中继标准输入/输入。",
```

Comment:
This looks weird when it doesn't follow our naming conventions.  Can you convert the first character to lower case when you generate everything?  Or maybe just rename the definitions in the options schema json?